build/build.xml

```diff
@@ -31,7 +31,7 @@
 
 	<!-- Ant task to append ZapAddOn.xml files to ZapVersions.xml -->
 	<taskdef classpathref="build.classpath" resource="zapversionstasks.properties"/>
-	
+
```

Comment:
Shouldnt make any whitespace changes - it makes the real changes harder to spot

Reply:
I isolated them in a specific commit when I started.
Did not want to make a PR just for that...